### DISCUSSION TITLE
Change dark theme color...

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6385] [Help] [answered]
Is there an easy way to change dark mode color..

If I remember correctly:

- the sidebar, table, model etc use `bg-gray-800`
- body background use `bg-gray-900`
- and border use `border-gray-700`

Suppose I don't want to publish view file then how can I change like sidebar, table etc `bg-slate-800` or another color and so on

**Comment by valpuia** [comment]
any help

**Reply by danharrin** [reply, marked as answer]
You can use CSS - we have many `filament-*` hook classes available for you to use
